src/components/sidebar/Sidebar.vue

```diff
@@ -1,5 +1,13 @@
 <template>
-  <va-sidebar :width="width" :minimized="minimized" :minimized-width="minimizedWidth" :animated="animated">
+  <va-sidebar
+    :width="width"
+    :minimized="minimized"
+    :minimized-width="minimizedWidth"
+    :animated="animated"
+    hover-color="#49A8FF"
```

Comment:
We probably can use color from config here instead. Not a huge prio though.

https://ui.vuestic.dev/services/colors-config#colors-config-with-other-services